english/utilize-wand/fork-rope-wand.md

```diff
@@ -0,0 +1,7 @@
+Letting out a light sigh, you grip your makeshift wand firmly and concentrate. Gathering your focus, you point your rope and fork wand to the door and firmly encant "Alohomora!"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.